src/java/com/puppetlabs/puppetserver/jruby/ScriptingContainer.java

```diff
@@ -0,0 +1,35 @@
+package com.puppetlabs.puppetserver.jruby;
+
+import org.jruby.embed.LocalContextScope;
+import org.jruby.embed.LocalVariableBehavior;
+
+/**
+ * An extension of the JRuby ScriptingContainer class which is
+ * slightly easier to use from Clojure
+ */
+public class ScriptingContainer
+        extends org.jruby.embed.ScriptingContainer {
+    public ScriptingContainer(LocalContextScope scope) {
+        super(scope);
+    }
+    public ScriptingContainer(LocalContextScope scope, LocalVariableBehavior behavior) {
+        super(scope, behavior);
+    }
+
+    /**
+     * This method delegates to a specific signature of #callMethod from the
```

Comment:
This makes sense.  Just curious whether you ran into a specific problem that led you to do this - i.e., you found out that our Clojure code was actually calling the wrong method - and what problem that may have caused.